docs/transactions/Basics.md

```diff
@@ -0,0 +1,45 @@
+# Transactions
+
+A transaction represents a movement of cash or holdings to/from a portfolio. When the transaction refers to a cash movement it will only contain a consideration. If the transaction is related to securities trading details about the security, price and quantity could also be included.
+
+The status of every order will first be “Matched” before moving to “Settled” or “Cancelled”.
+
+# Type of Transactions
+
+**Adjustment**: A transaction type used in orders to rectify a system error or an issue with the orders. E.g.: If the client was affected by postponed orders, we will then use the ‘adjustment’ transaction type for any alterations in the consideration or quantity.
+
+**Buy**: It represents the buying of a security. It will include the secutity ISIN code, the quantity and price of the trade and any other charges applied to the trade if any. When the trade has executed the transaction remains in matched state until it is settled
+
+**Sell**: It represents the selling of a security. It will include the secutity ISIN code, the quantity and price of the trade and any other charges applied to the trade if any. When the trade has executed the transaction remains in matched state until it is settled
+
+**Deposit**: A deposit made directly from the client’s bank account to us.
+
+**Withdrawal**: A payment out into the client’s bank account processed upon request.
+
+**Charge**: It is related to custody fees deducted in each portfolio as per instruction of each tenant.
+
+**Consolidation In/Out**: It's used to reflect some corporate actions operations.
+
+**Dividend**: It represent the payment of a dividend in cash, includes the secutity ISIN code.
```

Comment:
```suggestion
**Dividend**: Represents the payment of a dividend in cash.
```

If we want to detail the fields that are set with each type we should have a consistent way of describing that IMO.